portal-web/test/functional/com/liferay/portalweb/macros/FormsAdmin.macro

```diff
@@ -34,21 +34,30 @@ definition {
 			fieldName = "$url",
 			fieldValue = "${url}");
 
-		FormFields.editText(
-			fieldName = "$username",
-			fieldValue = "${username}");
+		if (isSet(username)) {
+			FormFields.editText(
+				fieldName = "$username",
+				fieldValue = "${username}");
 
-		var key_fieldName = "$password";
+			var key_fieldName = "$password";
 
-		if (IsElementPresent(locator1 = "FormFields#PASSWORD_FIELD")) {
-			FormFields.editPassword(
-				fieldName = "$password",
-				fieldValue = "${password}");
+			if (IsElementPresent(locator1 = "FormFields#PASSWORD_FIELD")) {
```

Comment:
Since the username and password are always occur in couple. I don't think it is necessary to add the condition.